keyboards/converter/HP_46010A/masks.csv

```diff
@@ -0,0 +1,112 @@
+menu,1,8,1
```

Comment:
@jackhumbert or @skullydazed Is this file okay, or do we want to remove it?

Reply:
i just added this for documentation tbh. Im hoping other HP 46010A boards are not different, as hp might have had different switch layouts, therefore the bit mask generated by the arduino program might be different.

Reply:
see  62d288a, should be updated now